js/controller.js

```diff
@@ -1116,11 +1116,25 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
      */
     onMultiAudioChanged: function(event, multiAudio) {
       if (!this.state.hideMultiAudioIcon) {
-        this.state.multiAudio = multiAudio;
+        if (this.containsMultiAudio(multiAudio)) {
+          this.state.multiAudio = multiAudio;
+        } else {
+          this.state.multiAudio = null;
+        }
+
         this.renderSkin();
       }
     },
 
+    /**
+     * Checks to see if we have multiple audio tracks
+     * @param multiAudio The multiAudio object that is fetched from the current video
+     * @returns {boolean} True if the provided object contains more than one track, false otherwise
+     */
+    containsMultiAudio: function(multiAudio) {
+      return !!(multiAudio && multiAudio.tracks && multiAudio.tracks.length > 1);
```

Comment:
Can you stick the result in a temp var and return that?

Reply:
Added